### PR TITLE
Custom sort calculation

### PR DESCRIPTION
**Pull request by sravfeyn**
http://manage.dimagi.com/default.asp?245496#1275860

screen shot 2017-01-20 at 12 04 48 pm

@calellowitz @ctsims

### CODE REVIEW

File: corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js [hunk 1]
```diff
@@ -164,14 +168,15 @@ hqDefine('app_manager/js/detail-screen-config.js', function () {
...
+        self.addSortRow = function (field, type, direction, display, notify, sortCalculation) {
             self.sortRows.push(new SortRow({
                 field: field,
                 type: type,
                 direction: direction,
                 display: display,
                 saveButton: saveButton,
-                properties: properties
+                properties: properties,
+                sortCalculation: sortCalculation
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js]: Missing trailing comma. (comma-dangle)

File: corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js [hunk 2]
```diff
@@ -1169,6 +1174,7 @@ hqDefine('app_manager/js/detail-screen-config.js', function () {
                                 type: row.type(),
                                 direction: row.direction(),
                                 display: row.display(),
+                                sort_calculation: row.sortCalculation()
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js]: Missing trailing comma. (comma-dangle)

**Comment by calellowitz** [comment]
looks good to me. i will wait on the lint errors unless you have a strong preference for merging now, but otherwise 👍

**Comment by czue** [comment]
linted

**Pull request merged**